db/migrate/20170223193040_add_workflow_ref_to_permission_templates.rb

```diff
@@ -0,0 +1,5 @@
+class AddWorkflowRefToPermissionTemplates < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :permission_templates, :workflow, foreign_key: true
```

Comment:
@mcritchlow it might impact these changes.  We can wait for it to be merged and run the tests again.